### PR TITLE
getting attribute from last response

### PR DESCRIPTION
**Pull request by mikeplavsky**

### CODE REVIEW

File: tests/RestApiSuite/TestGetAttributeFromLastReponse/content.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+!|post    |${URL}/users        |
+|FirstName|LastName|IsMafia|Age|
+|Tony     |Soprano |True   |22 |
+
+-!|script|RestTools|
+|check|get attribute|LastName|from last response|Soprano|
```
Comment (grigoryvasiliev) [inline tests/RestApiSuite/TestGetAttributeFromLastReponse/content.txt]: does it check post body which is sent or response which comes from this post?

Comment (mikeplavsky) [reply tests/RestApiSuite/TestGetAttributeFromLastReponse/content.txt]: why?

Comment (mikeplavsky) [reply tests/RestApiSuite/TestGetAttributeFromLastReponse/content.txt]: It just reads returned json if it is there

Comment (grigoryvasiliev) [reply tests/RestApiSuite/TestGetAttributeFromLastReponse/content.txt]: I was confused by this test as it posts data and then check result of last response - which means it check result of this post

Comment (mikeplavsky) [reply tests/RestApiSuite/TestGetAttributeFromLastReponse/content.txt]: This technique allows to store created object id in variable
`|$objectId=|get attribute|objectId|from last response|`

**Comment by grigoryvasiliev** [comment]
LGTM

**Pull request merged**